util/oec_utils.rb

```diff
@@ -28,4 +28,111 @@ def self.log_results(file, result, e = nil)
     end
   end
 
+  # Returns identifiers for a configured OEC test user
+  # @return [Hash]
+  def self.oec_user
+    {
+      first_name: @config['oec']['first_name'],
+      last_name: @config['oec']['last_name'],
+      uid: @config['oec']['uid'],
+      sis_id: @config['oec']['sis_id'],
+      email: @config['oec']['email']
+    }
+  end
+
+  # Returns the expected location of the folder containing sub-dirs and CSVs for the configured term code
+  # @return [String]
+  def self.term_folder
+    File.join(ENV['HOME'], "/OEC/#{@config['oec']['term']}")
+  end
+
+  # Returns departments that should be included in a given supervisors file but are missing
+  # @param file_name [String]
+  # @return [Array<String>]
+  def self.missing_supervisor_depts(file_name)
+    initial_csv = CSV.read(file_name, headers: true)
+    depts_present = []
+    initial_csv.each do |r|
+      depts_present << r['DEPT_NAME_1'] << r['DEPT_NAME_2'] << r['DEPT_NAME_3'] << r['DEPT_NAME_4'] << r['DEPT_NAME_5'] << r['DEPT_NAME_6'] << r['DEPT_NAME_7'] << r['DEPT_NAME_8'] << r['DEPT_NAME_9'] << r['DEPT_NAME_10']
```

Comment:
@pfarestveit You could shorten this a bit with a one-liner:
`1.upto(10) { |i| depts_present << r["DEPT_NAME_#{i}"] }`

Or write it like this and avoid the need to compact!
`1.upto(10) { |i| depts_present << r["DEPT_NAME_#{i}"] if r["DEPT_NAME_#{i}"]}`

Reply:
Oh thanks @paulkerschen .  i was wondering how to shorten that.